db/migrate/20250709075622_create_captain_scenarios.rb

```diff
@@ -0,0 +1,18 @@
+class CreateCaptainScenarios < ActiveRecord::Migration[7.1]
+  def change
+    create_table :captain_scenarios do |t|
+      t.string :title
+      t.text :description
+      t.text :instruction
+      t.jsonb :tools, default: []
+      t.boolean :enabled, default: true, null: false
+      t.references :assistant, null: false, foreign_key: { to_table: :captain_assistants }
```

Comment:
We don't add foreign keys. @sojan-official can confirm.

Reply:
You can define the model it should look for in the scenario.rb file. (Check assistant.rb for an example. )

```rb
has_many :documents, class_name: 'Captain::Document', dependent: :destroy_async
```